math/src/stableswap/tests/two_assets.rs

```diff
@@ -24,6 +24,12 @@ fn test_d_with_zero_reserves() {
 	assert_eq!(calculate_d::<D_ITERATIONS>(&reserves, 1), Some(0u128));
 }
 
+#[test]
+fn test_d_with_one_zero_reserves() {
```

Comment:
Maybe add the same test for `calculate_y`.

Reply:
calculate_y is tested within other tests where it is usually tested with more context.